cranelift/codegen/src/isa/riscv64/lower/isle.rs

```diff
@@ -76,6 +76,78 @@ impl generated_code::Context for IsleContext<'_, '_, MInst, Riscv64Backend> {
             | SignedLessThanOrEqual => ExtendOp::Signed,
         }
     }
+    fn normalize_fcvt_from_float(
```

Comment:
What do you think about moving this to ISLE? This seems like the type of code that would be really neatly written there.